src/FakeItEasy/Core/EventCall.cs

```diff
@@ -38,11 +38,16 @@ public static bool TryGetEventCall(IFakeObjectCall fakeObjectCall, [NotNullWhen(
             return true;
         }
 
-        public bool IsEventRegistration()
+        public bool IsEventSubscription()

```

Comment:
"Subscription" is a more common term than "Registration" for this. Also, I needed to add a method for unsubscription, and "Unregistration" sounded strange.